python/ray/tune/script_runner.py

```diff
@@ -123,6 +129,11 @@ def _init(self):
         self._last_reported_timestep = 0
         self._runner.start()
 
+    # Subclasses can override this to set the trainable func
+    # TODO(ekl) this isn't a very clean layering, we should refactor it
+    def _trainable_func(self):
```

Comment:
What's this supposed to do?

Reply:
I guess the only case this is used is in the script_runner? But I don't see any usage elsewhere...

Reply:
It's called in wrap_function().